recipe/0045-css/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: CSS in an Annotation
+id: 45
+layout: recipe
+tags: [style]
+summary: "Using an external CSS stylesheet in an annotation body, annotations can be styled in limited ways"
+viewers:
+ - Theseus
```

Comment:
I think this should be:

 - id: Theseus
   support: partial
   
As it doesn't support the target styling.

Reply:
The target styling is supported now on Theseus, but there is a CORS issue when loading the `style.css` file at the moment. https://theseusviewer.org/?iiif-content=https%3A%2F%2Fpreview.iiif.io%2Fcookbook%2F0045-css%2Frecipe%2F0045-css%2Fmanifest.json